Xabe.FFmpeg/Enums/VideoSize.cs

```diff
@@ -13,18 +13,36 @@ public class VideoSize
         /// </summary>
```

Comment:
I think you should overload ToString to return this property

Reply:
you are right